core/templates/I18nFooter.ts

```diff
@@ -50,7 +52,15 @@ angular.module('oppia').directive('i18nFooter', [
           };
           ctrl.$onInit = function() {
             ctrl.supportedSiteLanguages = SUPPORTED_SITE_LANGUAGES;
-
+            // Fetching the saved language code from cookie, because $translate
```

Comment:
You might want to note https://github.com/oppia/oppia/pull/9420/files (also, /cc @DubeySandeep).

Reply:
The changes made here shouldn't be affected by #9420.